core/base/supportedbases.go

```diff
@@ -76,26 +76,3 @@ func supportedInfoForType(path string, now time.Time, requestedBase Base, imageS
 
 	return supported, nil
 }
-
-// UbuntuVersions returns the ubuntu versions as a map.
-func UbuntuVersions(supported, esmSupported *bool) map[string]string {
-	return ubuntuVersions(supported, esmSupported, ubuntuSeries)
-}
```

Comment:
Glad to see this go. Once we have a hardcoded list, we don't need to ask these questions.